### PR TITLE
add and get trips

### PR DESCRIPTION
**Pull request by WGillian**

### CODE REVIEW

File: src/config/addTrip.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import fs from 'fs'
+import _ from 'lodash'
+
+export default (request, response) => {
+  if (!request.headers.token) {
```
Comment (richardgillzdev) [inline src/config/addTrip.js]: Do we need to check that the token is the same one we sent? I could send `token: sod.your.silly.tokens` and I would get a response

File: src/config/getTrips.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import fs from 'fs'
+import _ from 'lodash'
+
+export default (request, response) => {
+  if (!request.headers.token) {
```
Comment (richardgillzdev) [inline src/config/getTrips.js]: Same issue as above

Comment (richardgillzdev) [reply src/config/getTrips.js]: You should probable export the token from a file then use it as a single point of truth everywhere

File: src/config/getTrips.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  return fs.readFile('src/accounts/users.json', (readError, data) => {
+    if (readError) {
+      throw readError
+    }
+    const users = JSON.parse(data)
+    const matchedAccount = _.find(users, user => user.username === currentUser)
+    if (!matchedAccount) {
+      return response.status(401).json({ error: 'No account matches that username' })
+    }
+    return response.status(200).json({ success: 'Here are your trips!', trips: matchedAccount.trips })
```
Comment (richardgillzdev) [inline src/config/getTrips.js]: 'Here are your trips!' is frontend-y rather than server-y

File: src/config/addTrip.js (hunk 1 above)
Comment (WGillian) [reply src/config/addTrip.js]: hahaha

File: src/config/addTrip.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import fs from 'fs'
+import _ from 'lodash'
+import token from './token'
+
+export default (request, response) => {
+  if (request.headers.token !== token) {
+    return response.status(401).json({ error: 'No token found' })
```
Comment (richardgillzdev) [inline src/config/addTrip.js]: Error should now read along the lines of "Incorrect token"

File: src/config/getTrips.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import fs from 'fs'
+import _ from 'lodash'
+import token from './token'
+
+export default (request, response) => {
+  if (request.headers.token !== token) {
+    return response.status(401).json({ error: 'No token found' })
```
Comment (richardgillzdev) [inline src/config/getTrips.js]: Same comment as above

**Pull request merged**